src/dag/utils/StatusUtils.ts

```
@@ -0,0 +1,106 @@
+  }
+
+  /**
+   * Determines the appropriate icon for an artifact based on its type
+   */
+  static getArtifactIcon(artifactType: string, iconSvgs: { [name: string]: string }): string {
+    if (artifactType === 'ModelArtifact') {
+      return iconSvgs.dataflow || '';
+    }
+    return iconSvgs.database || '';
+  }
+}
```

Biome (1.9.4)

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)